source/install.md

```diff
@@ -0,0 +1,245 @@
+
+:::{hint}
+If you're running Mamba through MambaForge, you're configured to use the Conda Forge channel by default, and you don't need to pass the `-c conda-forge` argument to the commands in this page. Including this argument does no harm, so we've included it for the benefit of users that don't have this configuration.
+:::
+
+[Mamba]: https://mamba.readthedocs.io/en/latest/index.html
+[MambaForge]: https://github.com/conda-forge/miniforge#mambaforge
+[MambaForge repository]: https://github.com/conda-forge/miniforge#mambaforge
+[Mambaforge-MacOSX-x86_64]: https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-MacOSX-x86_64.sh
+[Mambaforge-Linux-x86_64]: https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-x86_64.sh
+
+(conda_init)=
+### To init or not to init
+
+After it's done installing, the MambaForge installer will ask you if you want it to initialize MambaForge. Initialization involves adding a section to your shell startup script that tells the shell where to find Conda/Mamba. Answering "no" will leave you startup scripts untouched, but you will need to run an additional command every time you want to use Conda/Mamba or a Conda environment. Answering "yes" will attempt to add a version of this command to your shell startup script, which generally makes Conda easier to use.
+
+If you do answer "yes", we recommend preventing Conda from activating the base environment in every new shell:
+
+```shell-session
+$ conda config --set auto_activate_base false
+```
+
+This will prevent any conflicts between the MambaForge Python installation and the Python expected by your system. It does not prevent you from using Conda/Mamba directly to manage or activate environments.
+
+(existing_conda)=
+### Installing Mamba in an existing Conda installation
+
+If you already have Conda installed, you don't need to install MambaForge; you can install Mamba on its own with:
+
+```shell
+conda install -c conda-forge mamba
+```
+
+Or you can simply replace any calls to `mamba` with an identical call to `conda`:
+
+```diff
+- mamba env create -c conda-forge -n openff -f environment.yaml
++ conda env create -c conda-forge -n openff -f environment.yaml
+```
+
+We recommend installing Mamba, as it can solve OpenFF environments in seconds where Conda would take minutes or hours.
+
+(managing_environments)=
+## Managing Environments
+
+Conda and Mamba install software into [virtual environments]. These environments include installations of some collection of software and all their dependencies in an isolated group that doesn't affect the software installed in the rest of your computer system. This means you can install software that's incompatible with what's already installed; you just put the new software in its own environment!
+
```

Comment:
Suggest adding something like

> Virtual environments are designed to be rapidly deployable, which also means they can quickly and safely be removed and re-created in a matter of seconds in case something goes wrong.

Personal preference to emphasize some smithed version of this, I regularly find that beginners waste days of their lives trying to fix weird states when blowing everything away and starting fresh takes a few minutes.

Reply:
Oh, maybe the following paragraph is a better location ...

Reply:
Done in 86c1c84